scripts/steam.zs

```diff
@@ -1,33 +1,134 @@
+
 mods.immersiveengineering.Refinery.addRecipe(<liquid:biodiesel> * 20, <liquid:plantoil> * 10, <liquid:ethanol> * 10, 2048);
+
 mods.immersivetechnology.PressurizedFluid.add(<fluid:water>);
 mods.immersivetechnology.PressurizedFluid.add(<fluid:distwater>);
 mods.immersivetechnology.PressurizedFluid.add(<fluid:high_pressure_steam>);
+
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*300, <fluid:low_pressure_steam>*120, <fluid:high_pressure_steam>*75, <fluid:exhaust_steam>*120, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*320, <fluid:preheated_water>*16, <fluid:high_pressure_steam>*80, <fluid:condensate_water>*16, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:corium_nak>*60, <fluid:condensate_water>*125, <fluid:corium_nak_hot>*60, <fluid:preheated_water>*125, 0, 1)
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*5120, <fluid:high_pressure_steam>*672, <fluid:super_critical_steam>*80, <fluid:crap_steam>*672, 0, 1)
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*320, <fluid:low_pressure_steam>*448, <fluid:super_critical_steam>*5, <fluid:exhaust_steam>*448, 0, 1)
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*1280, <fluid:preheated_water>*224, <fluid:super_critical_steam>*20, <fluid:condensate_water>*224, 0, 1)
+mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:moltensodium> * 40, <liquid:low_pressure_steam> * 30720,  <liquid:superheatedmoltensodium> * 40, <liquid:water> * 1920, 0, 10);
+//mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:moltensodium> * 40, <liquid:preheated_water> * 3840,  <liquid:superheatedmoltensodium> * 40, <liquid:condensate_water> * 3840, 0, 5); //Simplified Preheated Recipe
+mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:moltensodium> * 20, <liquid:preheated_water> * 11520,  <liquid:superheatedmoltensodium> * 20, <liquid:condensate_water> * 11520, 0, 15); //Nuclearcraft Standard Preheated Recipe
+mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:moltensodium> * 40, <liquid:high_pressure_steam> * 15360,  <liquid:superheatedmoltensodium> * 40, <liquid:preheated_water> * 3840, 0, 5);
+mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:moltensodium> * 1, <liquid:low_pressure_steam> * 4608,  <liquid:superheatedmoltensodium> * 1, <liquid:exhaust_steam> * 4608, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:moltensodium> * 25, <liquid:super_critical_steam> * 192,  <liquid:superheatedmoltensodium> * 25, <liquid:well_pressed_water> * 192, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<liquid:moltensodium> * 1, <liquid:high_pressure_steam> * 432,  <liquid:superheatedmoltensodium> * 1, <liquid:crap_steam> * 432, 0, 1);
+//mods.immersivetechnology.HeatExchanger.addRecipe(outSide, outTop,  inSide, inTop, 0, 1); //mnemonic reminder
```

Comment:
Please remove the commented lines.

Reply:
Are you sure about that? I left those comments in as alternatives in case you were not comfortable with the current recipes.

Reply:
I think going with the nuclearcraft recipe is a good choice because I want the power generation in the modpack to be compatible with nuclearcraft as much as possible. We can always tweak this more in the future if anything needs changing.